index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Larry Villegas Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+      </style>
+</head>
+<body>    
+    <main>
+        <div class="general-container">
+           <nav class="box-header">
```

Comment:
@osamaashraf6 
I rewrote the code so that it can be according to the recommendation you made.
![image](https://user-images.githubusercontent.com/110146573/220811251-a8fda909-5af1-47c4-9cb2-b1ccde2b64f1.png)
![image](https://user-images.githubusercontent.com/110146573/220811341-3822423e-d860-4e5a-97f3-0523e310bfc0.png)